packages/fx-core/templates/plugins/resource/cicd/github/README.md

```diff
@@ -0,0 +1,45 @@
+# How to use pre-cooked workflows on GitHub
+
+## Prerequisites
+- Teams app projects are version controlled by GitHub.
+- An M365 account. If you do not have M365 account, apply one from [M365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program)
```

Comment:
as my understanding, not all pipeline need M365 account. Only Provision and Publish need M365 account right?
we should make this clear to customer in the prerequsites.

Reply:
And also let customer know in prerequsites section that they should remove two-factor verification for their M365 account.